src/style/palette_ext.rs

```diff
@@ -0,0 +1,136 @@
+use num_traits::Float;
```

Comment:
That's awesome, I have been thinking about adding more color spaces for a long time.:+1: 

Really like this, thank you !